################################################################################
# PackageCompiler set-up: 2021-01-11T17:55:44.822
#

Julia Version 1.5.3
Commit 599f52c4c6 (2020-12-18 13:33 UTC)
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: AMD EPYC 7502 32-Core Processor
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, znver2)
Environment:
  JULIA_PKGEVAL = true
  JULIA_PKG_PRECOMPILE_AUTO = 0



################################################################################
# Installation: 2021-01-11T17:55:46.475
#

 Installing known registries into `~/.julia`
################################################                          66.8%######################################################################## 100.0%
      Added registry `General` to `~/.julia/registries/General`
  Resolving package versions...
  Installed Octo ──────────── v0.2.8
  Installed PackageCompiler ─ v1.2.5
Updating `~/.julia/environments/v1.5/Project.toml`
  [16905944] + Octo v0.2.8
  [9b87118b] + PackageCompiler v1.2.5
Updating `~/.julia/environments/v1.5/Manifest.toml`
  [16905944] + Octo v0.2.8
  [9b87118b] + PackageCompiler v1.2.5
  [2a0f44e3] + Base64
  [ade2ca70] + Dates
  [b77e0a4c] + InteractiveUtils
  [76f85450] + LibGit2
  [8f399da3] + Libdl
  [56ddb016] + Logging
  [d6f4376e] + Markdown
  [44cfe95a] + Pkg
  [de0858da] + Printf
  [3fa0cd96] + REPL
  [9a3f8284] + Random
  [ea8e919c] + SHA
  [9e88b42a] + Serialization
  [6462fe0b] + Sockets
  [cf7118a7] + UUIDs
  [4ec0a83e] + Unicode


################################################################################
# Compiling: 2021-01-11T17:55:52.141
#

[ Info: PackageCompiler: creating system image object file, this might take a while...
Generated system image is 150.821 MiB, compilation took 163 seconds


################################################################################
# PkgEval set-up: 2021-01-11T17:58:42.161
#

Julia Version 1.5.3
Commit 599f52c4c6 (2020-12-18 13:33 UTC)
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: AMD EPYC 7502 32-Core Processor
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, znver2)
Environment:
  JULIA_PKGEVAL = true
  JULIA_PKG_PRECOMPILE_AUTO = 0

Base.julia_cmd() = `/usr/local/julia/bin/julia -Cnative -J/cache/Octo-4op5nNl6.so -g1`


################################################################################
# Installation: 2021-01-11T17:58:43.987
#

 Installing known registries into `~/.julia`
######################################################################## 100.0%
      Added registry `General` to `~/.julia/registries/General`
  Resolving package versions...
  Installed Octo ─ v0.2.8
Updating `~/.julia/environments/v1.5/Project.toml`
  [16905944] + Octo v0.2.8
Updating `~/.julia/environments/v1.5/Manifest.toml`
  [16905944] + Octo v0.2.8
  [ade2ca70] + Dates
  [de0858da] + Printf
  [4ec0a83e] + Unicode


################################################################################
# Testing: 2021-01-11T17:58:48.116
#

    Testing Octo
Status `/tmp/jl_VpgUMa/Project.toml`
  [a93c6f00] DataFrames v0.21.8
  [6735fb99] Hive v0.4.0
  [6042db11] JDBC v0.5.0
  [ba5e3d4b] Jive v0.2.13
  [194296ae] LibPQ v1.6.2
  [aabc7e14] MariaDB_Connector_C_jll v3.1.6+0
  [39abe10b] MySQL v1.1.2
  [be6f12e9] ODBC v1.0.4
  [16905944] Octo v0.2.8
  [0aa819cd] SQLite v1.0.5
  [ade2ca70] Dates
  [8dfed614] Test
Status `/tmp/jl_VpgUMa/Manifest.toml`
  [79e6a3ab] Adapt v2.4.0
  [56f22d72] Artifacts v1.3.0
  [b99e7846] BinaryProvider v0.5.10
  [fa961155] CEnum v0.4.1
  [324d7699] CategoricalArrays v0.8.3
  [d360d2e6] ChainRulesCore v0.9.24
  [34da2185] Compat v3.25.0
  [e66e0078] CompilerSupportLibraries_jll v0.3.4+0
  [a10d1c49] DBInterface v2.2.0
  [9a962f9c] DataAPI v1.4.0
  [a93c6f00] DataFrames v0.21.8
  [864edb3b] DataStructures v0.18.8
  [e2d170a0] DataValueInterfaces v1.0.0
  [55939f99] DecFP v1.1.0
  [47200ebd] DecFP_jll v2.0.2+0
  [abce61dc] Decimals v0.4.1
  [ffbed154] DocStringExtensions v0.8.3
  [e2ba6199] ExprTools v0.1.3
  [8f5d6c58] EzXML v1.1.0
  [6735fb99] Hive v0.4.0
  [a303e19e] Infinity v0.2.3
  [d8418881] Intervals v1.5.0
  [41ab1584] InvertedIndices v1.0.0
  [c8e1da08] IterTools v1.3.0
  [82899510] IteratorInterfaceExtensions v1.0.0
  [6042db11] JDBC v0.5.0
  [692b3bcd] JLLWrappers v1.2.0
  [682c06a0] JSON v0.21.1
  [494afd89] JavaCall v0.7.6
  [ba5e3d4b] Jive v0.2.13
  [6f188dcb] LayerDicts v1.0.0
  [deac9b47] LibCURL_jll v7.70.0+2
  [194296ae] LibPQ v1.6.2
  [08be9ffa] LibPQ_jll v12.3.0+0
  [29816b5a] LibSSH2_jll v1.9.0+3
  [94ce4f54] Libiconv_jll v1.16.0+7
  [aabc7e14] MariaDB_Connector_C_jll v3.1.6+0
  [c8ffd9c3] MbedTLS_jll v2.16.8+1
  [f28f55f0] Memento v1.1.2
  [e1d29d7a] Missings v0.4.4
  [78c3b35d] Mocking v0.7.1
  [46d2c3a1] MuladdMacro v0.2.2
  [39abe10b] MySQL v1.1.2
  [be6f12e9] ODBC v1.0.4
  [16905944] Octo v0.2.8
  [6fe1bfb0] OffsetArrays v1.5.0
  [458c3c95] OpenSSL_jll v1.1.1+6
  [efe28fd5] OpenSpecFun_jll v0.5.3+4
  [bac558e1] OrderedCollections v1.3.2
  [69de0a69] Parsers v1.0.15
  [2dfb63ee] PooledArrays v0.5.3
  [3cdcf5f2] RecipesBase v1.1.1
  [189a3867] Reexport v0.2.0
  [ae029012] Requires v1.1.2
  [0aa819cd] SQLite v1.0.5
  [76ed43ae] SQLite_jll v3.34.0+0
  [a2af1166] SortingAlgorithms v0.3.1
  [276daf66] SpecialFunctions v1.2.1
  [856f2bd8] StructTypes v1.2.1
  [cea106d9] Syslogs v0.3.0
  [3783bdb8] TableTraits v1.0.0
  [bd369af6] Tables v1.2.2
  [8d9c9c80] Thrift v0.7.1
  [815b9798] ThriftJuliaCompiler_jll v0.11.0+0
  [f269a46b] TimeZones v1.5.3
  [ea10d353] WeakRefStrings v0.6.2
  [1b915085] WinReg v0.3.1
  [02c8fc9c] XML2_jll v2.9.10+3
  [83775a58] Zlib_jll v1.2.11+18
  [80337aba] iODBC_jll v3.52.13+1
  [8e850ede] nghttp2_jll v1.40.0+2
  [1841a5aa] unixODBC_jll v2.3.7+0
  [2a0f44e3] Base64
  [ade2ca70] Dates
  [8bb1440f] DelimitedFiles
  [8ba89e20] Distributed
  [7b1f6079] FileWatching
  [9fa8497b] Future
  [b77e0a4c] InteractiveUtils
  [76f85450] LibGit2
  [8f399da3] Libdl
  [37e2e46d] LinearAlgebra
  [56ddb016] Logging
  [d6f4376e] Markdown
  [a63ad114] Mmap
  [44cfe95a] Pkg
  [de0858da] Printf
  [3fa0cd96] REPL
  [9a3f8284] Random
  [ea8e919c] SHA
  [9e88b42a] Serialization
  [1a1011a3] SharedArrays
  [6462fe0b] Sockets
  [2f01184e] SparseArrays
  [10745b16] Statistics
  [8dfed614] Test
  [cf7118a7] UUIDs
  [4ec0a83e] Unicode
[4m1/35[24m adapters/mysql/execute_result_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/mysql/execute_result_test.jl:9
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.MySQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,Any,NTuple{6,Symbol},NamedTuple{(:username, :password, :hostname, :port, :db, :unix_socket),Tuple{String,String,String,Int64,String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/mysql/execute_result_test.jl:9
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/mysql/execute_result_test.jl:9
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/MySQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,Any,NTuple{6,Symbol},NamedTuple{(:username, :password, :hostname, :port, :db, :unix_socket),Tuple{String,String,String,Int64,String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/mysql/execute_result_test.jl:9
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m2/35[24m adapters/mysql/multiple_databases_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/mysql/multiple_databases_test.jl:11
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.MySQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,Any,NTuple{6,Symbol},NamedTuple{(:username, :password, :hostname, :port, :db, :unix_socket),Tuple{String,String,String,Int64,String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/mysql/multiple_databases_test.jl:11
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/mysql/multiple_databases_test.jl:11
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/MySQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,Any,NTuple{6,Symbol},NamedTuple{(:username, :password, :hostname, :port, :db, :unix_socket),Tuple{String,String,String,Int64,String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/mysql/multiple_databases_test.jl:11
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m3/35[24m adapters/mysql/placeholder_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/mysql/placeholder_test.jl:10
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.MySQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,Any,NTuple{6,Symbol},NamedTuple{(:username, :password, :hostname, :port, :db, :unix_socket),Tuple{String,String,String,Int64,String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/mysql/placeholder_test.jl:10
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/mysql/placeholder_test.jl:10
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/MySQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,Any,NTuple{6,Symbol},NamedTuple{(:username, :password, :hostname, :port, :db, :unix_socket),Tuple{String,String,String,Int64,String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/mysql/placeholder_test.jl:10
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m4/35[24m adapters/mysql/repo_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/mysql/repo_test.jl:9
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.MySQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,Any,NTuple{6,Symbol},NamedTuple{(:username, :password, :hostname, :port, :db, :unix_socket),Tuple{String,String,String,Int64,String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/mysql/repo_test.jl:9
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/mysql/repo_test.jl:9
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/MySQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,Any,NTuple{6,Symbol},NamedTuple{(:username, :password, :hostname, :port, :db, :unix_socket),Tuple{String,String,String,Int64,String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/mysql/repo_test.jl:9
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m5/35[24m adapters/mysql/structured_test.jl
    [32m[1mPass [22m[39m[32m11[39m  (3.45 seconds)
[4m6/35[24m adapters/odbc/mariadb_repo_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/odbc/mariadb_repo_test.jl:11
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.ODBC")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol},NamedTuple{(:dsn,),Tuple{String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/odbc/mariadb_repo_test.jl:11
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/odbc/mariadb_repo_test.jl:11
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/ODBC.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol},NamedTuple{(:dsn,),Tuple{String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/odbc/mariadb_repo_test.jl:11
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m7/35[24m adapters/postgresql/bigserial_pk_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/bigserial_pk_test.jl:8
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.PostgreSQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol,Symbol},NamedTuple{(:dbname, :user),Tuple{String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/bigserial_pk_test.jl:8
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/bigserial_pk_test.jl:8
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/PostgreSQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol,Symbol},NamedTuple{(:dbname, :user),Tuple{String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/bigserial_pk_test.jl:8
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m8/35[24m adapters/postgresql/boolean_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/boolean_test.jl:10
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.PostgreSQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol,Symbol},NamedTuple{(:dbname, :user),Tuple{String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/boolean_test.jl:10
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/boolean_test.jl:10
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/PostgreSQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol,Symbol},NamedTuple{(:dbname, :user),Tuple{String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/boolean_test.jl:10
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
    [32m[1mPass [22m[39m[32m1[39m  (0.43 seconds)
[4m9/35[24m adapters/postgresql/dates_test.jl
    [32m[1mPass [22m[39m[32m6[39m  (1.25 seconds)
[4m10/35[24m adapters/postgresql/execute_result_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/execute_result_test.jl:12
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.PostgreSQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol,Symbol},NamedTuple{(:dbname, :user),Tuple{String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/execute_result_test.jl:12
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/execute_result_test.jl:12
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/PostgreSQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol,Symbol},NamedTuple{(:dbname, :user),Tuple{String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/execute_result_test.jl:12
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m11/35[24m adapters/postgresql/functions_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/functions_test.jl:8
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.PostgreSQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol},NamedTuple{(:user,),Tuple{String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/functions_test.jl:8
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/functions_test.jl:8
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/PostgreSQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol},NamedTuple{(:user,),Tuple{String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/functions_test.jl:8
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m12/35[24m adapters/postgresql/multiple_databases_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/multiple_databases_test.jl:14
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.SQLite")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol},NamedTuple{(:dbfile,),Tuple{String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/multiple_databases_test.jl:14
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/multiple_databases_test.jl:14
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/SQLite.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol},NamedTuple{(:dbfile,),Tuple{String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/multiple_databases_test.jl:14
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m13/35[24m adapters/postgresql/null.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/null.jl:8
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.PostgreSQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol,Symbol},NamedTuple{(:dbname, :user),Tuple{String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/null.jl:8
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/null.jl:8
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/PostgreSQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol,Symbol},NamedTuple{(:dbname, :user),Tuple{String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/null.jl:8
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m14/35[24m adapters/postgresql/placeholder_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/placeholder_test.jl:8
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.PostgreSQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol,Symbol},NamedTuple{(:dbname, :user),Tuple{String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/placeholder_test.jl:8
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/placeholder_test.jl:8
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/PostgreSQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol,Symbol},NamedTuple{(:dbname, :user),Tuple{String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/placeholder_test.jl:8
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m15/35[24m adapters/postgresql/pool_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/pool_test.jl:12
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.PostgreSQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol,Symbol},NamedTuple{(:dbname, :user),Tuple{String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/pool_test.jl:12
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/pool_test.jl:12
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/PostgreSQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol,Symbol},NamedTuple{(:dbname, :user),Tuple{String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/pool_test.jl:12
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m16/35[24m adapters/postgresql/prepare_test.jl
    [32m[1mPass [22m[39m[32m2[39m  (0.94 seconds)
[4m17/35[24m adapters/postgresql/repo_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/repo_test.jl:14
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.PostgreSQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol,Symbol},NamedTuple{(:dbname, :user),Tuple{String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/repo_test.jl:14
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/repo_test.jl:14
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/PostgreSQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol,Symbol},NamedTuple{(:dbname, :user),Tuple{String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/repo_test.jl:14
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m18/35[24m adapters/postgresql/structured_test.jl
    [32m[1mPass [22m[39m[32m8[39m  (0.68 seconds)
[4m19/35[24m adapters/postgresql/subquery_test.jl
    [32m[1mPass [22m[39m[32m1[39m  (0.55 seconds)
[4m20/35[24m adapters/postgresql/transactions_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/transactions_test.jl:14
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.PostgreSQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol,Symbol},NamedTuple{(:dbname, :user),Tuple{String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/transactions_test.jl:14
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/transactions_test.jl:14
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/PostgreSQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol,Symbol},NamedTuple{(:dbname, :user),Tuple{String,String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/transactions_test.jl:14
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m21/35[24m adapters/postgresql/window_frames_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/window_frames_test.jl:19
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.PostgreSQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol},NamedTuple{(:user,),Tuple{String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/window_frames_test.jl:19
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/window_frames_test.jl:19
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/PostgreSQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol},NamedTuple{(:user,),Tuple{String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/postgresql/window_frames_test.jl:19
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
    [32m[1mPass [22m[39m[32m5[39m  (0.48 seconds)
[4m22/35[24m adapters/sql/structured_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/sql/structured_test.jl:32
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.SQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/sql/structured_test.jl:32
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/sql/structured_test.jl:32
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/SQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/sql/structured_test.jl:32
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
    [32m[1mPass [22m[39m[32m14[39m  (0.99 seconds)
[4m23/35[24m adapters/sqlite/execute_result_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/sqlite/execute_result_test.jl:14
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.SQLite")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol},NamedTuple{(:dbfile,),Tuple{String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/sqlite/execute_result_test.jl:14
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/sqlite/execute_result_test.jl:14
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/SQLite.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol},NamedTuple{(:dbfile,),Tuple{String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/sqlite/execute_result_test.jl:14
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m24/35[24m adapters/sqlite/repo_test.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/adapters/sqlite/repo_test.jl:16
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.SQLite")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol},NamedTuple{(:dbfile,),Tuple{String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/sqlite/repo_test.jl:16
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/adapters/sqlite/repo_test.jl:16
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/SQLite.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Symbol,String,Tuple{Symbol},NamedTuple{(:dbfile,),Tuple{String}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/adapters/sqlite/repo_test.jl:16
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m25/35[24m adapters/sqlite/structured_test.jl
    [32m[1mPass [22m[39m[32m12[39m  (0.95 seconds)
[4m26/35[24m octo/as.jl
    [32m[1mPass [22m[39m[32m2[39m  (0.12 seconds)
[4m27/35[24m octo/backends.jl
    [32m[1mPass [22m[39m[32m1[39m  (0.00 seconds)
[4m28/35[24m octo/predicates.jl
    [32m[1mPass [22m[39m[32m7[39m  (1.15 seconds)
[4m29/35[24m octo/pretty.jl
    [32m[1mPass [22m[39m[32m8[39m  (4.48 seconds)
[4m30/35[24m octo/raw.jl
    [32m[1mPass [22m[39m[32m1[39m  (0.15 seconds)
[4m31/35[24m octo/repo.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/octo/repo.jl:6
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.SQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/octo/repo.jl:6
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/octo/repo.jl:6
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/SQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/octo/repo.jl:6
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m32/35[24m octo/schema.jl
: Error During Test at /home/user/.julia/packages/Octo/9a5y7/test/octo/schema.jl:15
  Got exception outside of a @test
  LoadError: Octo.Backends.UnsupportedError("error on Octo.Adapters.SQL")
  Stacktrace:
   [1] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:18
   [2] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [3] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/octo/schema.jl:15
   [4] include(::String) at ./client.jl:457
   [5] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [6] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [7] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [8] include(::String) at ./client.jl:457
   [9] top-level scope at none:6
   [10] eval(::Module, ::Any) at ./boot.jl:331
   [11] exec_options(::Base.JLOptions) at ./client.jl:272
   [12] _start() at ./client.jl:506
  in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/octo/schema.jl:15
  caused by [exception 1]
  could not open file /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends/SQL.jl
  Stacktrace:
   [1] include(::Function, ::Module, ::String) at ./Base.jl:380
   [2] include at ./Base.jl:368 [inlined]
   [3] backend(::Module) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Backends.jl:16
   [4] connect(; adapter::Module, database::Nothing, multiple::Bool, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/pkgeval/.julia/packages/Octo/9a5y7/src/Repo.jl:82
   [5] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/octo/schema.jl:15
   [6] include(::String) at ./client.jl:457
   [7] normal_run(::String, ::Array{String,1}, ::Int64, ::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:469
   [8] runtests(::String; skip::Array{String,1}, node1::Array{Any,1}, targets::Array{String,1}, enable_distributed::Bool, stop_on_failure::Bool) at /home/user/.julia/packages/Jive/S1UMt/src/runtests.jl:87
   [9] top-level scope at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35
   [10] include(::String) at ./client.jl:457
   [11] top-level scope at none:6
   [12] eval(::Module, ::Any) at ./boot.jl:331
   [13] exec_options(::Base.JLOptions) at ./client.jl:272
   [14] _start() at ./client.jl:506
  
    [91m[1mError [22m[39m[91m1[39m
[4m33/35[24m octo/sql_functions.jl
    [32m[1mPass [22m[39m[32m6[39m  (0.19 seconds)
[4m34/35[24m octo/structured.jl
    [32m[1mPass [22m[39m[32m13[39m  (1.25 seconds)
[4m35/35[24m octo/subquery.jl
    [32m[1mPass [22m[39m[32m8[39m  (0.71 seconds)
[31m❗️  [39mTest run finished with 21 errors.  (43.81 seconds)
ERROR: LoadError: Test run finished with errors.
in expression starting at /home/user/.julia/packages/Octo/9a5y7/test/runtests.jl:35

PkgEval failed: Package Octo errored during testing
Stacktrace:
 [1] pkgerror(::String) at /workspace/srcdir/usr/share/julia/stdlib/v1.5/Pkg/src/Types.jl:52
 [2] test(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}; coverage::Bool, julia_args::Cmd, test_args::Cmd, test_fn::Nothing) at /workspace/srcdir/usr/share/julia/stdlib/v1.5/Pkg/src/Operations.jl:1578
 [3] test(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}; coverage::Bool, test_fn::Nothing, julia_args::Cmd, test_args::Cmd, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /workspace/srcdir/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:327
 [4] test(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /workspace/srcdir/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:314
 [5] #test#61 at /workspace/srcdir/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:67 [inlined]
 [6] test at /workspace/srcdir/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:67 [inlined]
 [7] #test#60 at /workspace/srcdir/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:66 [inlined]
 [8] test at /workspace/srcdir/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:66 [inlined]
 [9] test(::String; kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /workspace/srcdir/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:65
 [10] test(::String) at /workspace/srcdir/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:65
 [11] top-level scope at none:22
 [12] eval at ./boot.jl:331 [inlined]
 [13] eval(::Expr) at ./client.jl:467
 [14] top-level scope at none:1
 [15] eval(::Module, ::Any) at ./boot.jl:331
 [16] exec_options(::Base.JLOptions) at ./client.jl:272
 [17] _start() at ./client.jl:506


################################################################################
# PkgEval teardown: 2021-01-11T17:59:52.185
CPU usage: 68.96s (63.14s user, 5.94s kernel)

Network usage:
- eth0: 8.724 MiB received, 375.826 KiB sent

Raw statistics: {"blkio_stats":{"io_queue_recursive":[],"io_serviced_recursive":[{"major":259,"value":0,"op":"Read","minor":0},{"major":259,"value":39,"op":"Write","minor":0},{"major":259,"value":14,"op":"Sync","minor":0},{"major":259,"value":25,"op":"Async","minor":0},{"major":259,"value":0,"op":"Discard","minor":0},{"major":259,"value":39,"op":"Total","minor":0},{"major":253,"value":0,"op":"Read","minor":2},{"major":253,"value":39,"op":"Write","minor":2},{"major":253,"value":14,"op":"Sync","minor":2},{"major":253,"value":25,"op":"Async","minor":2},{"major":253,"value":0,"op":"Discard","minor":2},{"major":253,"value":39,"op":"Total","minor":2}],"io_time_recursive":[],"sectors_recursive":[],"io_service_bytes_recursive":[{"major":259,"value":0,"op":"Read","minor":0},{"major":259,"value":196608,"op":"Write","minor":0},{"major":259,"value":57344,"op":"Sync","minor":0},{"major":259,"value":139264,"op":"Async","minor":0},{"major":259,"value":0,"op":"Discard","minor":0},{"major":259,"value":196608,"op":"Total","minor":0},{"major":253,"value":0,"op":"Read","minor":2},{"major":253,"value":196608,"op":"Write","minor":2},{"major":253,"value":57344,"op":"Sync","minor":2},{"major":253,"value":139264,"op":"Async","minor":2},{"major":253,"value":0,"op":"Discard","minor":2},{"major":253,"value":196608,"op":"Total","minor":2}],"io_wait_time_recursive":[],"io_merged_recursive":[],"io_service_time_recursive":[]},"memory_stats":{"usage":348721152,"max_usage":1190330368,"stats":{"hierarchical_memory_limit":9223372036854771712,"pgmajfault":0,"total_rss":206639104,"inactive_file":135168,"inactive_anon":99753984,"total_pgfault":1849650,"total_cache":109031424,"total_rss_huge":0,"total_active_anon":215998464,"total_pgmajfault":0,"pgpgin":1266672,"rss_huge":0,"mapped_file":0,"total_inactive_file":135168,"dirty":135168,"total_unevictable":0,"total_dirty":135168,"writeback":0,"pgpgout":1189527,"hierarchical_memsw_limit":0,"cache":109031424,"active_file":135168,"pgfault":1849650,"total_mapped_file":0,"total_pgpgout":1189527,"total_writeback":0,"rss":206639104,"unevictable":0,"total_inactive_anon":99753984,"total_pgpgin":1266672,"active_anon":215998464,"total_active_file":135168},"limit":540965302272},"name":"/Octo-wIxQ6Ll6","networks":{"eth0":{"rx_bytes":9147659,"rx_packets":2665,"tx_packets":3080,"tx_bytes":384846,"tx_dropped":0,"rx_dropped":3,"tx_errors":0,"rx_errors":0}},"id":"cf609fe11364a7865759d19501b9b4adabba0b3e87b6378720c03758d0792a1b","preread":"0001-01-01T00:00:00Z","storage_stats":{},"precpu_stats":{"throttling_data":{"throttled_periods":0,"throttled_time":0,"periods":0},"cpu_usage":{"usage_in_usermode":0,"usage_in_kernelmode":0,"total_usage":0}},"pids_stats":{"current":44},"num_procs":0,"cpu_stats":{"throttling_data":{"throttled_periods":0,"throttled_time":0,"periods":0},"system_cpu_usage":4358068364522419,"online_cpus":128,"cpu_usage":{"usage_in_usermode":63140000000,"usage_in_kernelmode":5940000000,"total_usage":68957790881,"percpu_usage":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,68957777421,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,13460,0,0,0]}},"read":"2021-01-11T17:59:52.511437678Z"}